.changeset/fluffy-carrots-search.md

```diff
@@ -0,0 +1,25 @@
+---
+'astro': minor
+---
+
+Adds [`ShikiTransformer`](https://shiki.style/packages/transformers#shikijs-transformers) support to the [`<Code />`](https://docs.astro.build/en/reference/api-reference/#code-) component with a new `transformers` prop.
+
```

Comment:
```suggestion

Note that `transformers` only applies classes and you must provide your own CSS rules to target the elements of your code block.

```
I saw a note in the Shiki documentation about not applying styles, and I think it's helpful to mention that here! Maybe something like this?